sdk/cosmos/azure-cosmos/azure/cosmos/_timeout_failover_retry_policy.py

```diff
@@ -38,19 +37,13 @@ def __init__(self, connection_policy, global_endpoint_manager, *args):
         self.failover_retry_count = 0
         self.connection_policy = connection_policy
         self.request = args[0] if args else None
-
-        if self.request:
```

Comment:
We don't need the check for self.request any more? This is always assumed to be "true"?